### DISCUSSION TITLE
Crop Image

### DISCUSSION THREAD

**Discussion by kefilwe52** [dlemstra/Magick.NET#1697] [Help]
Hi @dlemstra ,

Could you please help me crop the document so that it starts and ends at the vertical lines, keeping only the content between them?

Crop the data from the left up to the vertical line and from the right up to the vertical line.

image.ColorSpace = ColorSpace.Gray;

    int leftMost = 0;
    int rightMost = image.Width;

    using (var verticalLines = image.Clone() as MagickImage)
    {
       
        verticalLines.Morphology(MorphologyMethod.Close, "rectangle:1x50");

        verticalLines.Threshold(new Percentage(50));
        
        
    
![image](https://github.com/user-attachments/assets/0f3da9cd-322f-4502-8435-919188755abb)

**Comment by dlemstra** [comment]
Normally you could use `.Trim()` for this but your image has some black pixels on the left side of the image so you will need to use a higher colorfuzz setting. This worked for me but you probably need to tweak this:

```C#
image.ColorFuzz = new Percentage(70);
image.Trim();
```
